### DISCUSSION TITLE
Getting error 23404 when trying to join gts on melonDS

### DISCUSSION THREAD

**Discussion by Laikar** [mm201/pkmnFoundations#90] [Q&A]
Tried joining GTS while playing pokemon white on [melonDS](http://melonds.kuribo64.net/), but i get error 23404 after some time, i have tried some times because i heard wifi is kinda janky on emulators, but never got anything other than 23404.
I tried on other Servers and got different errors.
Game says traffic volume is too high or service has been interrupted

**Comment by InternalLoss** [comment]
23404 means you're using a patched ROM on Kaeru's DNS, which (before the other day) did not support HTTP for the nas.nintendowifi.net proxying - please confirm that it now works :)
